### PR TITLE
Relocate repository to manubot/rootstock

### LINKED ISSUE #175: Migrating repo to the manubot organization
For those who aren't aware, we now control the @manubot GitHub username. We've set up a Manubot organization and would like to move all development to this organization.

In https://github.com/manubot/manubot/issues/94 / https://github.com/manubot/manubot/commit/1b3ac66de9220e0b7496128a4a45e444b6d4edb1 we moved the `greenelab/manubot` repo to `manubot/manbot`.

`manubot-rootstock` will be the most challenging repository to migrate, as we have to consider many factors. Here we'll discuss the migration plan.

Currently, we are thinking of moving `greenelab/manubot-rootstock` to `manubot/rootstock`. Here we'll catalog all the things that could break.

### PR DESCRIPTION
**Pull request by dhimmel**
Closes #175 
Supersedes #124 

Assumes https://github.com/greenelab/manubot-rootstock is relocated to https://github.com/manubot/rootstock by transferring the repo and then renaming it.

Doesn't include changes to files in `content/`, to prevent downstream conflicts. These can be changed in the next PR to touch the file, i.e. #169 if we remember.

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Cloning manubot-rootstock to create a new manuscript
+# Cloning rootstock to create a new manuscript
```
Comment (agitter) [inline SETUP.md]: Do you want to still include `Manubot` here?  Maybe `Cloning Manubot Rootstock...`?

File: SETUP.md [hunk 2]
```diff
@@ -168,9 +168,9 @@ git push origin master
 You should be good to go now.
 A good first step is to modify `content/metadata.yaml` with the relevant information for your manuscript.
 
-# Merging upstream manubot-rootstock changes
+# Merging upstream rootstock changes
```
Comment (agitter) [inline SETUP.md]: Same question as above in this section.  What is the most intuitive way to refer to this repository?  By the repository name or the more descriptive name?

Comment (dhimmel) [reply SETUP.md]: I think we can just call it rootstock. I updated the title in https://github.com/greenelab/manubot-rootstock/pull/180/commits/2ac7411890e82cf831e258a89b86ca93676cb9e6 so it's a bit more clear.

**Comment by dhimmel** [comment]
First relocated build current occurring at https://travis-ci.com/manubot/rootstock/builds/100700743

File: SETUP.md (hunk 2 above)
Comment (agitter) [reply SETUP.md]: Your changes were good clarifications

**Pull request merged**